include/boost/histogram/python/accumulators/ostream.hpp

```diff
@@ -38,51 +40,55 @@ handle_nonzero_width(std::basic_ostream<CharT, Traits> &os, const T &x) {
     return os;
 }
 
-} // namespace detail
+} // anonymous namespace
```

Comment:
The reason why boost libs use `detail` so much is because unnamed namespaces cannot be used, see previous comment.